edgelet/edgelet-http/src/lib.rs

```diff
@@ -70,6 +71,11 @@ const TCP_SCHEME: &str = "tcp";
 #[cfg(target_os = "linux")]
 const FD_SCHEME: &str = "fd";
 
+pub enum ConcurrencyThrottling {
+    MaxConcurrency(u32),
```

Comment:
I suggest calling it `Limited` instead of `MaxConcurrency`, just because on its own "max concurrency" sounds like "unlimited"

Reply:
Fair point, fixed.